src/Navigator/Browser.Navigator.fs

```diff
@@ -44,6 +44,52 @@ type NavigatorOnLine =
 type NavigatorUserMediaSuccessCallback = MediaStream -> unit
 type NavigatorUserMediaErrorCallback = MediaStreamError -> unit
 
+[<StringEnum; RequireQualifiedAccess>]
+type PermissionState =
+    | [<CompiledName("granted")>] Granted
+    | [<CompiledName("denied")>] Denied
+    | [<CompiledName("prompt")>] Prompt
+
+[<AllowNullLiteral>]
+type PermissionStatus =
+    abstract state: PermissionState
+    abstract status: PermissionState
+    abstract onchange: (Event -> 'Out) with get, set
+
+[<StringEnum; RequireQualifiedAccess>]
```

Comment:
You can pass a `CaseRules` argument to StringEnum, so you don't need to specify `CompiledName` for each case. Here it would be:

```fsharp
[<StringEnum(CaseRules.KebabCase); RequireQualifiedAccess>]
```

Reply:
Ok, I just realized this required an update of Fable.Core, but hopefully it should be fine. I'm using 3.1.5 which seems to be most widely installed.